src/pages/Dongwu/Main/Asides/Feed/Photo/PhotoCommentForm/PhotoCommentInput.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styles from './PhotoCommentInput.module.scss';
+
+class PhotoCommentInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      visibility: styles.visible,
+      input: React.createRef(),
```

Comment:
```suggestion
      input:{},
```
혹은
```suggestion
      input = React.createRef(),
```
렌더링에 연관되어 있는 값이면 state로 사용하시고 그렇지 않다면 멤버변수에 ref로 걸어주세요. 언제 ref 사용해야 하는지 공식문서에서 찾아보시고 댓글로 남겨주세요

Reply:
해당 suggestion 사항인 Input React DOM 접근에 대해 공부하고 보니 부끄러워져서 삭제했습니다.

먼저, createRef의 경우 현재 제가 사용할 만한 사항이 딱히 없다고 판단이 들었습니다.

공식문서의 Ref와 DOM 을 참고했는데, 

- 포커스, 텍스트 선택영역, 혹은 미디어의 재생 관리
- 애니메이션 직접 실행
- 3rd-party-DOM-library 와 React의 혼용

위 세 사항이 바람직한 상황이라고 한 것에서 input을 텍스트 선택영역으로 볼 것인가에 대해 생각했으나, 아니라고 결론짓게 되었습니다.

사실 이전에도 해당 내용을 인지하고 있었으나, 포커스라고 명시된 부분에 자연스레 넘어가게 되었던 것 같습니다.

그러나 제가 생각한 포커스가 아니었습니다.

이후 변경된 코드는 PhotoForm - PhotoInput의 state-props 구조를 고민해본 결과, PhotoForm state의 commentText를 PhotoCommentInput의 input value로 할당하기로 결정했습니다.

자세한 구조는 이후 변경된 코드로 보시는게 나을 것 같아 다음 PR에 작성을 명시하도록 하겠습니다.